### PR TITLE
Add manifest for "hotfix" plugin

### PR DESCRIPTION
**Pull request by steadysupply**
Closes #492 (and follows on from it)
Documentation at https://steady.supply/git/kubectl-hotfix
Source code available with `git clone https://steady.supply/git/kubectl-hotfix`

### CODE REVIEW

File: plugins/hotfix.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        path: src:/app
+        target:
+          - label: service in (web, api)
+          - label: service=postgres
+            path: src/database:/app/database
+          - label: service=chrome
+            shell: bash
+          - label: service=static
+            include:
+              - "*.{html,css,js}"
```
Comment (corneliusweig) [inline plugins/hotfix.yaml]: This should be moved into a dedicated help for this command like `hotfix --help`. The description is meant to explain _what_ a plugin does and not how to configure it.

File: plugins/hotfix.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hotfix
```
Comment (corneliusweig) [inline plugins/hotfix.yaml]: `hotfix` is not very descriptive. I think your name should mention `sync` or `transfer`, like your short description does.

File: plugins/hotfix.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+      uri: https://steady.supply/git/kubectl-hotfix/release/v0.0.3/kubectl-hotfix-v0.0.3-darwin-amd64.tar.gz
+      sha256: "8a879f5256630866bfe033bae70b66256065c817c193c3e31ca9dd0aa78530bc"
+      files:
+        - from: "*"
+          to:  "."
```
Comment (corneliusweig) [inline plugins/hotfix.yaml]: This is the default so you can also omit the copy instructions.

File: plugins/hotfix.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hotfix
+spec:
+  version: "v0.0.3"
+  shortDescription: dumb file transfer to your labelled pods
+  homepage: https://steady.supply/git/kubectl-hotfix
+  description: |
```
Comment (corneliusweig) [inline plugins/hotfix.yaml]: This description reads more like a man page.

Please focus on what your plugin does and why a user would need it. The details you give here should be moved to the commands `--help` output.

File: plugins/hotfix.yaml (hunk 3 above)
Comment (steadysupply) [reply plugins/hotfix.yaml]: Thanks

Comment (steadysupply) [reply plugins/hotfix.yaml]: Wait, I can omit `to: "."` or the whole `files` key?

Comment (ahmetb) [reply plugins/hotfix.yaml]: Whole files key.

**Comment by corneliusweig** [comment]
@steadysupply What's your opinion on the above points? If you don't intend to follow up, I'd like to close this issue.

**Pull request closed without merging**